src/OpenTK.NT/Native/User32/Delegates/TimerProc.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Runtime.InteropServices;
+
+using DWORD = System.UInt32;
+using HWND = System.IntPtr;
+using UINT_PTR = System.IntPtr;
```

Comment:
Should this be UIntPtr, or is it a pointer to an uint?

Reply:
A `UIntPtr`. I just forgot to change it